pkg/table/index.go

```diff
@@ -102,7 +102,7 @@ type Index interface {
 	// FetchValues fetched index column values in a row.
 	// Param columns is a reused buffer, if it is not nil, FetchValues will fill the index values in it,
 	// and return the buffer, if it is nil, FetchValues will allocate the buffer instead.
-	FetchValues(row []types.Datum, columns []types.Datum) ([]types.Datum, error)
+	FetchValues(ctx MutateContext, row []types.Datum, columns []types.Datum) ([]types.Datum, error)
```

Comment:
It's strange to pass `MutateContext` to `FetchValues` as an argument because this method  does not mutate any data.